indico/modules/events/registration/badges.py

```diff
@@ -129,7 +130,8 @@ def _draw_badge(self, canvas, person, template, tpl_data, pos_x, pos_y):
             item_data = {'item': item, 'text': text, 'pos_x': pos_x, 'pos_y': pos_y}
             for update in values_from_signal(
                 signals.event.designer.draw_item_on_badge.send(person['registration'], items=items, height=self.height,
-                                                               width=self.width, data=item_data, person=person),
+                                                               width=self.width, data=item_data, person=person,
+                                                               template_data=template_data),
```

Comment:
Why not pas `tpl_data` directly?

Reply:
`tpl_data` has too many things. All of the items, dimension in pixel and in in cm.

Reply:
Why is that "too many things"? You still have access tow hat you need. It doesn't cost anything to pass the rest as well...

Reply:
`items` is already passed as a dedicated parameter. So it's a bit confusing passing them again in a different structure. But maybe just for me so I don't mind passing the whole `tpl_data`. Shall I update then?

Reply:
Yeah, I think it's better to pass the full `tpl_data`. And the items in there are unsorted, so it still makes sense to have `items` separately